typescript/web/src/connectors/undo-store/effects/create-label-class-and-create-label.ts

```diff
@@ -72,7 +72,7 @@ export const createCreateLabelClassAndCreateLabelEffect = (
     const { data } = await client.mutate({
       mutation: CREATE_LABEL_MUTATION,
       variables: { data: createLabelInputs },
-      refetchQueries: ["countLabelsOfDataset"],
+      refetchQueries: ["CountLabelsOfDatasetQuery"],
```

Comment:
Not sure if it comes from here, but when creating or deleting a label, a warning appears

```
Unknown query named "CountLabelsOfDatasetQuery" requested in refetchQueries options.include array
```

Reply:
isn't that the query for exports, that we stopped launching automatically every time we get on the dataset page?